imports/ui/components/newTask/newTask.js

```diff
@@ -3,14 +3,35 @@ import { Tasks } from '../../../api/api.js';
 
 import './newTask.html';
 
-Template.newTask.rendered=function() {
-  $('#duedate').datepicker();
+Template.datepicker.rendered = function() {
+  this.$('.datepicker').datepicker({
+    autoclose: true,
+  });
+};
+
+Template.newTask.rendered = function() {
```

Comment:
same as before, 1 template per file, 1 template's js per file. Again we'll keep this for now